lib/storage/s3.js

```diff
@@ -22,12 +22,14 @@ class S3 extends require('./base') {
   }
 
   put (key, data) {
-    let params = this._params(key, {Body: data})
-    if (data instanceof Stream) {
```

Comment:
also, I'm going to make CI use S3 instead of the filesystem which doesn't test this code